src/module/skb/skb.rs

```diff
@@ -37,26 +30,17 @@ impl Collector for SkbCollector {
         Ok(SkbCollector {})
     }
 
-    fn name(&self) -> &'static str {
-        SKB_COLLECTOR
-    }
-
     fn known_kernel_types(&self) -> Option<Vec<&'static str>> {
         Some(vec!["struct sk_buff *"])
     }
 
     fn register_cli(&self, cmd: &mut DynamicCommand) -> Result<()> {
-        cmd.register_module::<SkbCollectorArgs>(SKB_COLLECTOR)
+        cmd.register_module::<SkbCollectorArgs>(ModuleId::Skb.to_str())
```

Comment:
I think we can make `DynamicCommand` accept a `ModuleId` directly instead of a str